vendor/engines/secure_rooms/app/models/card_reader.rb

```diff
@@ -0,0 +1,5 @@
+class CardReader < ActiveRecord::Base
+
+   belongs_to :control_device
```

Comment:
I just realized I could probably just run rubocop on `engines/secure_rooms` and spot these much more easily 😝